internal/cmd/swap/cmd.go

```diff
@@ -221,13 +221,13 @@ func determineSwapData(args []string, repo execute.OpenRepoResult) (data swapDat
 	if err != nil || exit {
 		return data, exit, err
 	}
-	branchNameToSwap := gitdomain.NewLocalBranchName(slice.FirstElementOr(args, branchesSnapshot.Active.String()))
-	branchToSwapInfo, hasBranchToSwapInfo := branchesSnapshot.Branches.FindByLocalName(branchNameToSwap).Get()
+	currentBranch := gitdomain.NewLocalBranchName(slice.FirstElementOr(args, branchesSnapshot.Active.String()))
```

Comment:
This naming makes a lot more sense. `branchToSwap` was ambiguous, since two branches are swapping places!